commands/admin.js

```diff
@@ -115,20 +126,20 @@ exports.promote = function(message, args) {
     }
     var id_rx = /^<@([0-9]+)>$/g;
     var id = id_rx.exec(args[0]);
-    console.log(id);
-    id = id ? id[1] : args[0]
-         console.log(id);
+    id = id ? id[1] : args[0];
     var user = global.util.getUser({id: id});
+    logger.info(`Locating user ${id}`);
     if(user === undefined || global.userList[user.id]>=2){
-        message.channel.send(`Cannot promote that user!`)
+        message.channel.send(`Cannot promote that user!`);
         return;
     }
+    logger.info(`Promoting ${user.username}`);
     global.database.promoteUser(user).then(
         function(results){
-            global.util.updateUserPermissions(message.channel, `Successfully promoted ${args[0]}`);
+            global.util.updateUserPermissions(message.channel, `Successfully promoted ${user}`);

```

Comment:
Can we modify this error message to be more specific to the promotion. Either to user level numerically or with an enum that reference the name compared to level in DB.